### ISSUE TITLE
Questions about t-SNE plot.

### ISSUE THREAD

**Issue by omg777** [Jiaolong/self-supervised-da#2]
Hi, 
Thanks for you code sharing.

I have a question about t-SNE plot.
Which features are used to draw t-SNE plot?
features after encoder or after classifier ?

**Comment by Jiaolong** [comment]
We use the features after the classifier, i.e., the class logits.

**Comment by deepai-lab** [comment]
Hi @Jiaolong,

Thank you for your nice work. Can you please share your t-SNE plotting code?
